src/azure-cli/azure/cli/command_modules/network/tests/hybrid_2018_03_01/test_dns_commands.py

```diff
@@ -141,7 +142,7 @@ def test_dns(self, resource_group):
             self.cmd('network dns record-set {0} remove-record -g {{rg}} --zone-name {{zone}} --record-set-name myrs{0} {1}'.format(t, args[t]))
 
         self.cmd('network dns record-set a show -n myrsa -g {rg} --zone-name {zone}',
-                 checks=self.check('length(arecords)', 1))
+                 checks=self.check('length(aRecords)', 1))
```

Comment:
It's a breaking change in response. So I think you should add it in release notes.

Reply:
Good catch and I'll  add it to release notes.